app/models/response.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: responses
+#
+#  id            :bigint           not null, primary key
+#  array_value   :text             is an Array
```

Comment:
Whats an example of an array response?

Reply:
I could also see "date" being a response field - not sure if thats already captured here

Reply:
An array would be multiple choices selected for one question, perhaps?

Date isn't captured per se but I agree. I'll try to add it in.